managed_vms/endpoints/src/main/java/com/example/managedvms/endpoints/AuthInfoServlet.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.managedvms.endpoints;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Base64;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+/**
+ * A servlet that returns authentication information.
+ * See swagger.yaml for authentication mechanisms (e.g. JWT tokens, Google ID token).
```

Comment:
Maybe add a link to the endpoints docs describing the `X-Endpoint-API-UserInfo` header? Also is the output always one of these? https://jwt.io/ Maybe link to that, too.